RFC/src/Glossary.md

```diff
@@ -80,7 +80,7 @@ The complete state of the blockchain at a specific block height. This means a pr
 [broadcast strategy]: #broadcaststrategy "Strategy used for broadcasting messages in a peer-to-peer network"
 
 A strategy for propagating messages amongst nodes in a peer-to-peer network. Example implementations of
-`BroadcastStrategy` include the Gossip protocol, Dandelion and flood fill.
+`BroadcastStrategy` include the Gossip and flood fill.
```

Comment:
```suggestion
`BroadcastStrategy` include the Gossip protocol and flood fill.
```
It should either be, gossip or the gossip protocol.

Reply:
The term protocol is used a little loosely but [wikipedia](https://www.wikiwand.com/en/Gossip_protocol) uses it just as loosely so I'll re-add it :)